src/algebra/big_operators/finprod.lean

```diff
@@ -416,6 +418,19 @@ equals the product of `f i` divided by the product over `g i`. -/
 by simp only [div_eq_mul_inv, finprod_mul_distrib hf ((mul_support_inv g).symm.rec hg),
               finprod_inv_distrib]
 
+lemma finprod_inv_distrib₀ {G : Type*} [comm_group_with_zero G] (f : α → G) :
+  ∏ᶠ x, (f x)⁻¹ = (∏ᶠ x, f x)⁻¹ :=
+begin
+  classical,
+  have h_supp : mul_support (λ i : α, (f i)⁻¹) = mul_support f ,
+  { simp only [mul_support, ne.def, inv_eq_one₀] },
+  rw finprod_def,
+  split_ifs with hf,
+  { rw [finprod_def, ← h_supp, dif_pos hf],
+    exact finset.prod_inv_distrib' },
+  { rw [finprod_def, ← h_supp, dif_neg hf, inv_one] }
+end
+
```

Comment:
Can you move `finprod_inv_distrib` next to this lemma?

Reply:
Also, I think if you define the missing `mul_equiv.inv₀` you can prove this in one line just like how the `comm_group` version is proved.

Reply:
Ok, I'll see if I can rewrite it that way.